### DISCUSSION TITLE
I cannot comprehend how to get --listen to work, and I thought I have done everything I can think of.

### DISCUSSION THREAD

**Discussion by Grandvichy** [oobabooga/text-generation-webui#6378] [Q&A]
I've been trying my best to locally connect from my phone to the webui running on my PC, I've hit a few walls, but now I'm at the point where I literally do not comprehend what else there is to be done to get it to work.

I ran webui with, command loaded --listen just fine, according to the command lines, webui was running locally on one of the port.
I wasn't able to connect to it from my phone initially so I went and opened the port in the firewall, and still that is not enough. 
Genuinely, what is missing? 
There's no tutorial for this as far I as searched through the 'wiki', I don't understand how no one had this issue when I looked it up. 

Update: Tried to update via update_wizard_windows.bat and now webui doesn't even launch anymore because files were unmerged?
Hoping there's a wizard out there will be able to help figure this out. I didn't think I was this bad at technology but I'm genuinely stumped.
![unmerged files](https://github.com/user-attachments/assets/034928a9-2174-4571-8a76-a51f92215d51)

**Comment by Kaszebe** [comment]
Based on what you just said---I assume you want to run Oobabooga at home and then use your phone to connect to the web server when you are away from your home?

Are you trying to connect to the internal IP address of your home network from your phone? In other words---when you are testing this out is your phone on cellular or is it on WiFi and connected to your home network?

If you are using cellular, you cannot reach your web server on the internal IP address (e.g. 192.168.X.XX:7860). You will need to get the actual ip address of the server/computer where Ooobabooga is running on and then connect.

And of course make the necessary changes in your firewall.

I have also found that using an AI can help with issues like this. I hate to reccomend the free version of ChatGPT or Claude3....but try going there and asking it your tech question. Not suggesting you not ask for help in this forum---but rather just giving you a tip on what I do before I come on here to ask questions.

**Reply by Grandvichy** [reply]
Hey, thanks for helping out. I'm trying to just connect locally, not away from home.
Both devices are connected to the same network. PC via ethernet and the phone on WiFi.
The changes I made to the firewall was adding an inbound rule to open the port.
The idea of asking AI is actually good, I just forgot that chatGPT is a thing

**Reply by CallMeRive** [reply]
I'd try to figure out is there really connectivity between your phone and your PC. Some routers can isolate Wi-Fi devices from accessing other devices on the network. Try pinging your PC from your phone.
Then I'd try to disable windows firewall _temporarily_ to test if it's a problem...
